src/Tyler.jl

```diff
@@ -1,29 +1,63 @@
 module Tyler
 
-using Makie
-using LinearAlgebra
+import Colors
+import Extents
+import GeoInterface
+import GeometryBasics
+import HTTP
+import ImageMagick
+import LinearAlgebra
+import LRUCache
+import Makie
+import OrderedCollections
+import ThreadSafeDicts
+
+using Colors: RGB, N0f8
+using Extents: Extent
+using GeometryBasics: GLTriangleFace, Point2f, Vec2f, Rect2f, Rect2, Rect, decompose, decompose_uv
+using LRUCache: LRU
 using MapTiles: MapTiles, Tile, TileGrid, web_mercator, wgs84, CoordinateReferenceSystemFormat
+using Makie: Observable, Figure, Axis, RGBAf, on, isopen, meta, mesh!, translate!, scale!
+using OrderedCollections: OrderedSet
+using ThreadSafeDicts: ThreadSafeDict
 using TileProviders: TileProviders, AbstractProvider, geturl, min_zoom, max_zoom
-using Colors
-using Colors: N0f8
-using LRUCache
-using GeometryBasics
-using GeometryBasics: GLTriangleFace, decompose_uv
-using Extents
-using GeoInterface
-using ThreadSafeDicts
-using OrderedCollections
-using HTTP
-using ImageMagick
 
 include("for_interpolations.jl")
-using Tyler.ForInterpolations: tile2positions, Interpolator
 
 const TileImage = Matrix{RGB{N0f8}}
 
+"""
+    Map
+
+    Map(extent, [extent_crs=wgs84]; kw...)
+
+Tylers object, it plots tiles onto a Makie.jl `Axis`,
+downloading and plotting more as you zoom and pan.
+
+# Arguments
+
+-`extent`: the initual extent of the map, as a `GeometryBasics.Rect`
+    or an `Extents.Extent` in the projection of `extent_crs`.
+-`extent_crs`: Any `GeoFormatTypes` compatible crs, the default is wsg84.
+
+# Keywords
+
+-`resolution`: The figure resolution.
+-`figure`: an existing `Makie.Figure` object.
+-`crs`: The providers coordinate reference system.
```

Comment:
What is the relation between the extent_crs and the crs keyword? Is one going to be projected onto the other?

Reply:
`extent_crs` just lets you enter an extent in a different projection. `crs` is the actual projection of the tiles.